src/app/utils/tests/TestUtils.spec.ts

```diff
@@ -173,6 +173,14 @@ export class SimpleComponentTestUtils<T> {
         expect(element).withContext(elementName + ' should exist').toBeTruthy();
         return element;
     }
+    public expectElementToBeEnabled(elementName: string): void {
+        const element: DebugElement = this.findElement(elementName);
+        expect(element.nativeElement.disabled).withContext(elementName + ' should be enabled').toBeFalsy();
+    }
+    public expectElementToBeDisabled(elementName: string): void {
+        const element: DebugElement = this.findElement(elementName);
+        expect(element.nativeElement.disabled).withContext(elementName + ' should be disabled').toBeTruthy();
+    }
```

Comment:
simple component devrait pas hériter de component, ou l'inverse, pour éviter cette duplication ?

Reply:
oui, depuis un moment ! Ticket "TestUtils: refactor" ?

Reply:
Ticket "TestUtils: use inheritance in ComponentTestUtils"